packages/babel-config/src/plugins/babel-plugin-redwood-routes-auto-loader.ts

```diff
@@ -225,7 +253,7 @@ export default function (
 
 function prerenderLoaderImpl(
   prerender: boolean,
```

Comment:
```suggestion
  forPrerender: boolean,
```
Should we just go ahead and make this one consistent with all the rest of the renaming that's going on?